02-express-tutorial/answers/final/005-all-static.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.static('./Public'));
```

Comment:
You probably want to use `path.resolve` again, as the current approach only works if the file is executed from the `answers` directory. 

By using `path.resolve(__dirname, '../Public')`, you can execute the file from any directory and it will work. (Try executing the file from different locations to see what I mean.)

Reply:
Corrected this. The Public folder is outside of the "Final" folder so the "../Public" needs to be included as you mentioned in order to step back into the parent directory where the public folder is located.